module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -101,6 +102,25 @@ Read the [`EnhancedVolcano` package documentation](https://rdrr.io/bioc/Enhanced
 
 
 
+## `DESeq2` and other differential expression functions
```

Comment:
Maybe? 
```suggestion
## `DESeq2` and pseudobulking functions
```

Reply:
👍 
changing this to `pseudo-bulking` for consistency